packages/apps/plugins/plugin-layout/src/LayoutPlugin.tsx

```diff
@@ -279,7 +279,10 @@ export const LayoutPlugin = ({
                 },
               };
 
-        const attended = useMemo(() => new Set(location.active ? [location.active] : []), [location.active]);
+        const attended = useMemo(
+          () => new Set(location.active ? [firstMainId(location.active)] : []),
+          [location.active],
+        );
```

Comment:
Would attention / attended state be better suited to being declared as a signal alongside `settings`, `layout` and `location`?

Reply:
Yes, I’m just now getting to LayoutPlugin’s implementation, this was just a formatting change.